content/influxdb/cloud/account-management/limits.md

```diff
@@ -70,15 +66,16 @@ To write historical data older than 30 days, retain data for more than 30 days,
   - Unlimited checks
   - Unlimited notification rules
   - Unlimited notification endpoints for [all endpoints](/flux/v0.x/tags/notification-endpoints/)
+- **Storage**: Set your retention period to unlimited or up to 1 year by [updating a bucket’s retention period in the InfluxDB UI](/influxdb/cloud/organizations/buckets/update-bucket/#update-a-buckets-retention-period-in-the-influxdb-ui), or [set a custom retention period](/influxdb/cloud/organizations/buckets/update-bucket/#update-a-buckets-retention-period) using the [`influx` CLI](influxdb/cloud/reference/cli/influx/).
 
 ## Global limits
 
-InfluxDB Cloud global system limits cannot be adjusted and apply to all accounts.
-These hard limits are typically dictated by the capabilities of the underlying InfluxDB Cloud infrastructure.
+InfluxDB Cloud applies global (non-adjustable) system limits to all accounts, which protects the InfluxDB Cloud infrastructure for all users. As the service continues to evolve, we'll continue to review these global limits and adjust them as appropriate.
+
 Limits include:
 
 - Write request limits:
-  - 50 MB maximum HTTP request batch size (compressed or uncompressed--defined in the `Content-Type` header)
+  - 5 MB maximum HTTP request batch size (compressed or uncompressed--defined in the `Content-Type` header)
```

Comment:
I believe this is actually the `Content-Encoding` header that signifies the body is compressed or not

Reply:
thank you, @glinton